pkg/ulog/log.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019 the u-root Authors. All rights reserved
```

Comment:
shouldn't this be log_linux.go? very linux specific.

Reply:
done.